app/(authenticated)/dashboard/_components/board.tsx

```diff
@@ -0,0 +1,32 @@
+import Profiles from './profiles';
+import { Leaderboard } from './database';
+
+export default function Board() {
```

Comment:
Well, it is not wrong to create component with this pattern but into our project, we are following a pattern, we are using **arrow function** to create a component to keep sync with others, check other components. So, I would like to recommend you to follow the pattern.

Reply:
Also I would like to rename this component from `Board` to `Leaderboard`.

Reply:
fixed